houndigrade/test_cli.py

```diff
@@ -69,10 +69,6 @@ def mock_glob_side_effect(pattern):
 
         self.assertEqual(result.exit_code, 0)
         self.assertIn('"cloud": "aws"', result.output)
-        self.assertIn(
```

Comment:
Can we replace this assert with another one instead of simply removing it?

Reply:
@ilyavw Sure! The only reason I didn't is because we removed this section of the json & we didn't add anything in its place. Do you have a suggestion of what you would like me to assert?

Reply:
I think adding an assert that verifies that we indeed have the key with the above image_id present (`ami-123456789`) and maybe a couple more asserts to verify that we found redhat on the first partition and centos on the second one. Thanks!

Reply:
@ilyavw Added some asserts. See if this is what you had in mind. Thank you!